verification/curator-service/ui/src/components/landing-page/SignInForm.tsx

```diff
@@ -131,11 +134,10 @@ export default function SignInForm() {
             formik.resetForm();
         };
         // eslint-disable-next-line
-    }, [registrationScreenOn]);
+    }, [setRegistrationScreenOn]);
```

Comment:
Why are you passing `setRegistrationScreenOn` as `useEffect` dependency? I think it is unnecessary